docs/conf.py

```diff
@@ -115,6 +115,21 @@
     "^/dev[0-9a-f]{9}$"
 ]
 
+linkcheck_ignore = [
+    r"^./android/gradle.html$",
+    r"^./iOS/xcode.html$",
+    r"^./linux/appimage.html$",
+    r"^./linux/flatpak.html$",
+    r"^./linux/system.html$",
+    r"^./macOS/app.html$",
+    r"^./macOS/xcode.html$",
+    r"^./web/static.html$",
+    r"^./windows/app.html$",
+    r"^./windows/visualstudio.html$",
+    r"^https://github.com/beeware/briefcase/issues/\d+$",
+    r"^https://github.com/beeware/briefcase/pull/\d+$",
```

Comment:
I always forget to note when something is "intentionally provocative" and I fully expect push back...as I did here.

The only alternative I really have in mind is setting [`linkcheck_workers`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_workers) to something larger than five (5). It may at least run faster then.....as long as GitHub doesn't start returning 429s or something.

Reply:
Move fast and break things, right? :-)

In this case, I think the risk profile is pretty small. The release notes are updated once per release; it's not like a "normal" documentation page where links *could* be updated in any pull request (or an external documentation source could change). I think I can live with the ignore, since it will speed things up for *most* docs builds, and as releases build up, the list of links is going to get absurd - to the point where even N-fold parallelisation won't help (especially when you consider that we get a handful of links added every Monday because of dependabot).